packages/cozy-client/package.json

```diff
@@ -20,7 +20,7 @@
     "@types/lodash": "^4.14.170",
     "btoa": "^1.2.1",
     "cozy-stack-client": "^60.6.0",
-    "date-fns": "2.29.3",
+    "date-fns": "^2.30.0",
```

Comment:
I think you should fix the version like we did in cozy-ui because we can have this packages in multiples lib (cozy-client, cozy-ui...) and in the app at the same time 🤔